doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -974,32 +974,42 @@ Generation of induction principles with ``Scheme``
    .. insertprodn scheme_kind sort_family
 
    .. prodn::
-      scheme_kind ::= Equality for @reference
-      | {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
+      scheme_kind ::= {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
       sort_family ::= Set
       | Prop
       | SProp
       | Type
 
-  A high-level tool for automatically generating
-  (possibly mutual) :term:`induction principles <induction principle>`
-  for given types and sorts.
-  Each :n:`@reference` is a different inductive type identifier belonging to
-  the same package of mutual inductive definitions.
-  The command generates the :n:`@ident`\s as mutually recursive
-  definitions. Each term :n:`@ident` proves a general principle of mutual
-  induction for objects in type :n:`@reference`.
+   Generates (possibly mutual) :term:`induction principles <induction principle>`
+   for given types and sorts. Each :n:`@reference` is a different inductive type
+   identifier belonging to the same package of mutual inductive definitions. The
+   command generates the :n:`@ident`\s as mutually recursive definitions. Each
+   term :n:`@ident` proves a general principle of mutual induction for objects
+   in type :n:`@reference`.
 
-  :n:`@ident`
-    The name of the scheme. If not provided, the scheme name will be determined automatically
-    from the sorts involved.
+   :n:`@ident`
+      The name of the scheme. If not provided, the scheme name will be determined
+      automatically from the sorts involved.
+
+   The following :n:`@scheme_kind`\s generate induction principles with
+   given properties:
+
+   =================== =========== ===========
+    :n:`@scheme_kind`   Dependent   Recursive
+   =================== =========== ===========
+    :n:`Induction`         Yes         Yes
+    :n:`Minimality`        No          Yes
+    :n:`Elimination`       Yes         No
+    :n:`Case`              No          No
```

Comment:
With two boolean columns, I'd usually expect the rightmost one to be Yes No Yes No.
I suggest either swaping the Dependent and Recursive columns or reordering the entries.  If you
reorder the entries, it would be a nice touch to use that order in the mlg file

Reply:
I think the order makes sense like this. I will swap the columns.